src/components/Navbar.tsx

```diff
@@ -1,4 +1,14 @@
+import classNames from 'classnames';
+import { NavLink } from 'react-router-dom';
+
 export const Navbar = () => {
+  const activeLinkClass = ({ isActive }: { isActive: boolean }) => (
+    classNames('navbar-item',
+      {
+        'has-background-grey-lighter': isActive,
+      })
+  );
```

Comment:
It is better to define this function outside the component so that it is not created every time a new render is performed.